data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/constant/ErrorMessage.java

```diff
@@ -0,0 +1,14 @@
+package com.scalar.db.dataloader.cli.constant;
+
+public class ErrorMessage {
```

Comment:
Currently, we manage all the error messages of exceptions in `CoreError`:
https://github.com/scalar-labs/scalardb/blob/a6cac67efe6644899e6ed2937e29e5e344ca577f/core/src/main/java/com/scalar/db/common/error/CoreError.java

Can we move these error messages to `CoreError`?

Reply:
Thank you. I have moved all the errors to the ScalarDB `CoreError` class.